distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -735,4 +738,38 @@ public void clearCatalog() {
     public void clearCache() {
         console.runCommand(CLEAR_CACHE);
     }
+
```

Comment:
❗️ We should add JavaDoc to those methods to explain what they do and how to use them.